<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SVPropNetworkHeatExchangeWidget</class>
 <widget class="QWidget" name="SVPropNetworkHeatExchangeWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>521</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="groupBoxHeatExchange">
     <property name="title">
      <string>Heat Exchange</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_10">
      <item row="3" column="1">
       <widget class="QtExt::ValidatingLineEdit" name="lineEditHXTransferCoefficient"/>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="labelTemperature">
        <property name="text">
         <string>Temperature [C]</string>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="labelDataFile">
        <property name="text">
         <string>Data spline</string>
        </property>
       </widget>
      </item>
      <item row="5" column="1">
       <widget class="QComboBox" name="comboBoxZoneId"/>
      </item>
      <item row="4" column="1">
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLabel" name="labelHeatExchangeSpline">
          <property name="text">
           <string>TextLabel</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="toolButtonHeatExchangeSpline">
          <property name="text">
           <string>...</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="5" column="0">
       <widget class="QLabel" name="labelZoneId">
        <property name="text">
         <string>Zone Id</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QComboBox" name="comboBoxHeatExchangeType"/>
      </item>
      <item row="1" column="1">
       <widget class="QtExt::ValidatingLineEdit" name="lineEditHeatFlux"/>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="labelHXTransferCoefficient">
        <property name="text">
         <string>External heat transfer coefficient [W/mÂ²K]</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="labelHeatFlux">
        <property name="text">
         <string>Heat flux [W]</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QtExt::ValidatingLineEdit" name="lineEditTemperature"/>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="labelHeatFlux_2">
        <property name="text">
         <string>Type</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBoxCurrentHeatExchange">
     <property name="title">
      <string>Heat Exchange Types used in current project</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_9">
      <item row="0" column="0" colspan="2">
       <widget class="QTableWidget" name="tableWidgetHeatExchange"/>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer_5">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>60</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QtExt::ValidatingLineEdit</class>
   <extends>QLineEdit</extends>
   <header location="global">QtExt_ValidatingLineEdit.h</header>
   <slots>
    <signal>editingFinishedSuccessfully()</signal>
   </slots>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
